examples/cd_clf_cifar10.ipynb

```diff
@@ -18,9 +18,17 @@
   },
```

Comment:
<p>Any reason to do just one type of corruption? Should update the accompanying text and delete the commented line if so.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Will fix.